src/Teams/Controllers/MultipleAnswerQuestionController.cs

```diff
@@ -14,28 +14,74 @@ namespace Teams.Controllers
 {
     public class MultipleAnswerQuestionController : Controller
     {
-        public MultipleAnswerQuestionController(IMultipleAnswerQuestionRepository questionRepository)
+        private readonly IApplicationDbContext _dB;
+        private readonly IMultipleAnswerQuestionRepository _questionRepository;
+
+        public MultipleAnswerQuestionController(IMultipleAnswerQuestionRepository questionRepository, IApplicationDbContext db)
         {
-            this.questionRepository = questionRepository;
+            this._questionRepository = questionRepository;
+            _dB = db;
         }
-        private readonly IMultipleAnswerQuestionRepository questionRepository;
         [HttpGet]
```

Comment:
Please add Route attribute for every endpoint. It's very declarative and redable approach.